packages/web/components/Player/Section/PlayerMeToken.tsx

```diff
@@ -0,0 +1,648 @@
+                        ? collateralTokenData.symbol
+                        : symbol
+                    }
+                    for
+                    ${
+                      transactionType === 'mint'
+                        ? symbol
+                        : collateralTokenData.symbol
+                    }`
+                    : 'Approve Tokens'}
+                </MetaButton>
+              ) : (
+                <MetaButton mx="auto" mt="1rem" onClick={handleSwitchToMainnet}>
+                  Switch Chain
+                </MetaButton>
+              )}
+            </Flex>
+          </TabPanel>
+          <TabPanel>
+            <Flex direction="column" justifyItems="center">
+              <Box width="sm" bg="white" borderRadius="lg">
+                <Flex justify="space-between" align="center" p="2">
+                  <Box>
+                    <Input
+                      backgroundColor={'#fffff'}
+                      color={'black'}
+                      border={'none'}
+                      value={amount}
+                      p={2}
+                      type="text"
+                      onChange={(e) => handleSetSpendAmount(e.target.value)}
+                      name="Amount"
+                    />
+                    <Text
+                      color="gray"
+                      fontSize={'12'}
+                      textAlign={'left'}
+                      ml={2.5}
+                    >
+                      {roundNumber(meTokenData.balance)}
+                    </Text>
+                  </Box>
+                  <Wrap align="center">
+                    <Image
+                      src={profilePicture}
+                      height="36px"
+                      width="36px"
+                      borderRadius={50}
+                      mx="auto"
+                      alt="profile picture"
+                    />
+                    <Text color="black">{symbol}</Text>
+                  </Wrap>
+                </Flex>
+                <hr />
+                <Flex justify="space-between" align="center" p="2">
+                  <Text
+                    display="flex"
+                    alignItems="center"
+                    gap="5px"
+                    color="black"
+                  >
+                    meToken Value
+                    <HiOutlineInformationCircle />
+                  </Text>
+                  <Text color="black">$ {roundNumber(previewAmount)}</Text>
+                </Flex>
+              </Box>
+              {chainId === '0x1' ? (
+                <MetaButton
+                  mx="auto"
+                  mt="1rem"
+                  onClick={handleSpendMeTokens}
+                  disabled={loading}
+                >
+                  {loading ? 'Loading..' : `Spend ${symbol}`}
+                </MetaButton>
+              ) : (
+                <MetaButton mx="auto" mt="1rem" onClick={handleSwitchToMainnet}>
```

Comment:
I think there is an existing component that does this. See `SwitchNetworkButton`

Reply:
Used SwitchNetworkButton